lib/layout/data_grid.dart

```diff
@@ -474,6 +474,54 @@ class DataGridState extends WidgetState<DataGrid>
     setState(() {});
   }
 
+  void customSort(List dataList, String? sortKey, bool isAscendingOrder) {
+    if (sortKey == null) return;
+    final lists = JsonPath(sortKey).read(dataList).toList();
+    if (lists.length >= 2) {
+      dataList.asMap().forEach((index, value) {
```

Comment:
does this even work? You loop through each item in the list then sort the entire grid? Please verify correctness throroughly with unit tests. 
Either way this logic is off. Please refactor

Reply:
@vusters  Something is wrong here, looks like my code is missing. Not sure, I'll look check and get back to you